pylot/component_creator.py

```diff
@@ -452,10 +452,12 @@ def add_planning(goal_location, pose_stream, prediction_stream, camera_stream,
     return waypoints_stream
 
 
-def add_control(pose_stream, waypoints_stream):
```

Comment:
Can you please put the ground_vehicle_id_stream last, and make it optional? In this way we don't have to update lincoln.py and any other scripts that use add_control. Plus ground_vehicle_id_stream only appplies to carla_auto_pilot, so it shouldn't be first.